bigquery_etl/query_scheduling/generate_airflow_dags.py

```diff
@@ -93,6 +94,17 @@ def get_dags(project_id, dags_config):
                     logging.error(f"Error processing task for query {query_file}")
                     raise e
                 else:
+                    if CHECKS_FILE in files:
+                        checks_file = os.path.join(root, CHECKS_FILE)
+                        checks_task = copy.deepcopy(
+                            Task.of_dq_check(checks_file, dag_collection=dag_collection)
+                        )
+                        tasks.append(checks_task)
+                        task_ref = TaskRef(
+                            dag_name=task.dag_name,
+                            task_id=task.task_name,
+                        )
+                        checks_task.upstream_dependencies.append(task_ref)
```

Comment:
we were using `depends_on` previously but should have used `upstream_dependencies`. `depends_on` is generally using for defining dependencies on Airflow tasks that live in telemetry-airflow (or anything not coming from bigquery-etl)